.travis.yml

```diff
@@ -26,7 +30,7 @@ before_deploy:
   
 deploy:
   provider: script
-  script: heroku container:release --app todoapp-jb web 
+  script: heroku container:release web --app todoapp-jb && chmod a+rwx ./webhook.sh && bash ./webhook.sh
```

Comment:
We can get rid of the Heroku releases now if we want - no need to keep both around.

Reply:
Removed heroku releases